### PR TITLE
Separate the adblock preferences from the advanced features

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch separates the adblock preferences from the advanced features.

Refs: #1039

### CODE REVIEW

File: res/xml/banana_extension_preferences.xml [hunk 1]
```diff
@@ -38,6 +38,10 @@
         android:summary="@string/clear_browsing_data_summary" />
 
     <PreferenceCategory android:title="@string/detail_settings" />
+    <Preference
```
Comment (romandev) [inline res/xml/banana_extension_preferences.xml:41]: This should be in AdvancedFeatureSettings.

File: res/values-ko/strings.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <string name="add_secret_tab">시크릿 탭 열기</string>
     <string name="add_to_home">홈 화면에 추가</string>
     <string name="advanced_features">고급 기능</string>
-    <string name="advanced_features_summary">광고차단, 번역, 데이터 절약, 외부 다운로드 매니저...</string>
```
Comment (romandev) [inline res/values-ko/strings.xml:14]: Unnecessary change

File: res/values/strings.xml [hunk 3]
```diff
@@ -11,7 +11,7 @@
     <string name="add_secret_tab">Add Secret tab</string>
     <string name="add_to_home">Add to Home</string>
     <string name="advanced_features">Advanced Features</string>
-    <string name="advanced_features_summary">AdBlock, Translate, Data reduction mode, External download manager...</string>
```
Comment (romandev) [inline res/values/strings.xml:14]: Unnecessary change

File: res/xml/banana_extension_preferences.xml (hunk 1 above)
Comment (jdragonbae) [reply res/xml/banana_extension_preferences.xml]: Thanks for the review! I miss it.

File: res/values-ko/strings.xml (hunk 2 above)
Comment (jdragonbae) [reply res/values-ko/strings.xml]: Done

File: res/values/strings.xml (hunk 3 above)
Comment (jdragonbae) [reply res/values/strings.xml]: Done

**Review by romandev — APPROVED** [review]
lgtm
